packages/contracts-bedrock/scripts/go-ffi/trie.go

```diff
@@ -1,15 +1,16 @@
-package trie
+package main
 
 import (
 	"crypto/rand"
 	"fmt"
 	"log"
 	"math/big"
+	"os"
 
 	"github.com/ethereum/go-ethereum/accounts/abi"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
-	"github.com/ethereum/go-ethereum/ethdb/memorydb"
+	"github.com/ethereum/go-ethereum/core/rawdb"
```

Comment:
just curious why, speed?

Reply:
Updated api - `test-case-generator` was its own go module, and the new `trie` api wants a full implementation of the `ethdb.Database` interface, which `memorydb` doesn't provide.